include/tscore/ink_config.h.cmake.in

```diff
@@ -104,6 +104,7 @@ const int DEFAULT_STACKSIZE = @DEFAULT_STACK_SIZE@;
 #define TS_MAX_HOST_NAME_LEN @TS_MAX_HOST_NAME_LEN@
 
 /* Feature Flags */
+#cmakedefine01 HAVE_BROTLI_ENCODE_H
```

Comment:
Move this up with the other HAVE_ and make it just `#cmakedefine`

Reply:
There are preprocessor directives that require it to be 0 or 1.